src/redux/todo/reducers/operations.js

```diff
@@ -18,6 +18,9 @@ export const todoOperationsReducer = (state = defaultState, action) =>{
         case DELALLTASKS:
             state = [];
             return [];
+        case REMOVETODO:
+            const filteredTodos = state.filter((todo)=>todo.id !== action.payload);
+            return filteredTodos;
```

Comment:
No need to make constant here, its better like this:
`return state.filter(todo => todo.id !== action.payload);`